functions/src/definitions/checkerHandlers.js

```diff
@@ -195,8 +195,11 @@ async function onFactCheckerYes(voteRequestPath, from, platform = "whatsapp") {
   const messageRef = voteRequestRef.parent.parent
   const messageSnap = await messageRef.get()
   const message = messageSnap.data()
+  const latestInstanceRef = messageSnap.get("latestInstance")
+  const latestInstanceSnap = await latestInstanceRef.get()
+  const latestType = latestInstanceSnap.get("type") ?? "text"
   let res
-  switch (message.type) {
+  switch (latestType) {
```

Comment:
To prevent function from crashing, will be good to handle the `default` case. It looks like in our case, the logic for `text` can be used as default.

Reply:
Done